Filter active controls in profile by tags (#5596)
chef/inspec: docs-chef-io/content/inspec/cli.md

    Read configuration from JSON file (`-` reads from stdin).
* ``--controls=one two three``
    A list of control names to run, or a list of /regexes/ to match against control names. Ignore all other tests.
* ``--tags=one two three``

>     A list of tags names that are part of controls to filter and run controls, or a list of /regexes/ to match against tags names of controls. Ignore all other tests. And when tag on the control is a hashmap, it only uses values of hashmap for filtering controls.

Reviewer: ```suggestion
    A list of tags or a list of regular expressions that match tags. `exec` will run controls referenced by the listed or matching tags.
```

Author: @IanMadd I feel we should also include in doc that if tags are of type hash then it matches the value of an hash not a key to bring more clarity. I mentioned this sentence for that purpose "And when tag on the control is a hash map, it only uses values of hashmap for filtering controls." Can we put this in more clear way in doc ? What are your suggestions?

Reviewer: Maybe something like this: 

A list of tags, a list of regular expressions that match tags, or a hash map where each value is a tag. 
`exec` will run controls referenced by the listed or matching tags.

* ``--controls=one two three``
    A list of controls to include. Ignore all other tests.
* ``--tags=one two three``

>     A list of tags to filter controls and include only those. Ignore all other tests.

Reviewer: ```suggestion
    A list of tags that reference certain controls. Other controls are ignored.
```
